<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1347</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>authentification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Fenêtre principale */
QDialog {
	
	background-image: url(:/img/images/104016.jpg);

    background-color: #2c3e50; /* Bleu foncé */
    color: #ecf0f1; /* Blanc cassé */
    border-radius: 10px;
    padding: 20px;
}

/* Champs de saisie (QLineEdit) */
QLineEdit {
    background-color: #34495e; /* Bleu-gris */
    color: #ecf0f1; /* Blanc cassé */
    border: 2px solid rgb(0, 74, 173); /* Bleu */
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}

QLineEdit:focus {
    border: 2px solid rgb(0, 74, 173); /* Bleu */
    background-color: #3b5998; /* Bleu plus clair */
}

/* Bouton de connexion (QPushButton) */
QPushButton {
    background-color: rgb(0, 74, 173); /* Bleu */
    color: #ecf0f1; /* Blanc cassé */
    border: none;
    border-radius: 5px;
    padding: 10px;
    font-size: 16px;
    font-weight: bold;
    transition: background-color 0.3s ease;
}

QPushButton:hover {
    background-color: #004a9d; /* Bleu légèrement plus foncé */
}

QPushButton:pressed {
    background-color: #003a7d; /* Bleu encore plus foncé */
}

/* Labels (QLabel) */
QLabel {
    color: #ecf0f1; /* Blanc cassé */
    font-size: 14px;
    font-weight: bold;
}

/* Placeholder text dans QLineEdit */
QLineEdit[placeholderText] {
    color: #95a5a6; /* Gris clair */
}

/* MessageBox */
QMessageBox {
    background-color: #34495e; /* Bleu-gris */
    color: #ecf0f1; /* Blanc cassé */
    border-radius: 10px;
}

</string>
  </property>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>350</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mdp">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>420</y>
     <width>331</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>mot de passe</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_in">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>490</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>-20</y>
     <width>601</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/image/icons/Black_and_White_Blue_Modern_Letter_W_Logo-removebg-preview.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>711</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/img/images/startup-construction-development-3d-thin-line-art-style-design-concept-isometric-illustration.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFacialRecognition">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>550</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconnaissance _faciale</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCameraPreview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>691</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelMessage">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>640</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
    font-size: 14px;
    font-weight: bold;
    margin-bottom: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="emojie">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>331</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black;
    font-size: 14px;
    font-weight: bold;
    margin-bottom: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>email</zorder>
  <zorder>mdp</zorder>
  <zorder>sign_in</zorder>
  <zorder>label_2</zorder>
  <zorder>btnFacialRecognition</zorder>
  <zorder>labelCameraPreview</zorder>
  <zorder>labelMessage</zorder>
  <zorder>emojie</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
